liqd_lunch/frontend/src/RestaurantsList.js

```diff
@@ -77,6 +83,16 @@ class  RestaurantsList  extends  Component {
         });
     }
 
+    timeForLunch() {
+        var now = new Date();
```

Comment:
Please lets stop using `var`. `let` if it gets redeclared, in all other cases, like here: `const`